src/main/python/cfn_sphere/aws/kms.py

```diff
@@ -22,7 +21,7 @@ def decrypt(self, encrypted_value):
 
     def encrypt(self, key_id, cleartext_string):
         try:
-            response = self.client.encrypt(KeyId=key_id, Plaintext=binary_type(cleartext_string, 'utf-8'))
```

Comment:
Tested with Python 2.7 and 3.4. I also downgraded boto3 to version 1.3 and botocore to 1.4.0 to see if it was a recent change in boto, but the test case keeps working.